manifests/config.pp

```diff
@@ -80,6 +82,23 @@
     group => 'root',
     mode  => '0644',
   }
+  # UID and GID map are stored in separate files and included in the config.
+  $_cvmfs_id_map_file_prefix = '/etc/cvmfs/config.d/default'
```

Comment:
One question?

Would you ever want different maps for different repos ?

/cc @jblomer

Reply:
Yes, I think that would be the norm. In particular, I'd expect this feature only deployed for some special cases, e.g. container images.

Reply:
In my opinion, yes — in our case, we want the maps exclusively for our local container repository. 
(since we don't have `CLAIM_OWNERSHIP` for any other repository, specifying it globally would also not harm in our case)

I would expect other use cases to exist where repositories from different sources may need different mapping, but Jakob surely has a better overview :smile: .